src/methods/randomshuffle.jl

```diff
@@ -7,9 +7,16 @@ This method destroys any linear
 correlation in the signal, but preserves its amplitude distribution.
 """
 struct RandomShuffle <: Surrogate end
-surrogate(x, ::RandomShuffle) = randomshuffle(x)
+function surrogate(x, rs::RandomShuffle)
+    sg = surrogenerator(x, rs)
+    sg()
+end
```

Comment:
@kahaaga I don't understand why you defined this method? It gives you literally nothing over the existing definition of `surrogate`.

You only had to delete the previous old method. If you think about how Julia's multiple dispatch works, it actually creates exactly the method you wrote based on the existing `surrogate` definition in the file `api.jl`.

Reply:
You are absolutely right, this is unneccesary. Removed.